src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import core.basesyntax.ball.Ball;
+import core.basesyntax.ball.Color;
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        int indexColor = new Random().nextInt(Color.values().length);
```

Comment:
`Random` must be a class field
```suggestion
        int indexColor = random.nextInt(Color.values().length);
```